assets/stylesheets/mobile.scss

```diff
@@ -0,0 +1,3 @@
+.alert.alert-collective-alert {
+  font-size: 14px;
```

Comment:
It could be a topic for the next call. Nevertheless, I want to raise the fact that we should enforce the use of relative units. In order to strive for accessibility in the software we deliver.

Reply:
Sure, I can change it now if you want, using rems right?

Reply:
Yes, rems. The thing is that I do not know how Discourse is built-in so I do not know it makes sense or not. In any case, it should be the standard tho.

Reply:
Ok cool, I’ll research how discourse does it before making this change, i think they use px everywhere so it could be a problem to use rems right away.

We could enforce this with a stylelint configuration or something later on.